Sources/NIOCore/EventLoop.swift

```diff
@@ -478,8 +479,22 @@ public struct NIODeadline: Equatable, Hashable, NIOSendable {
         self._uptimeNanoseconds = nanoseconds
     }
 
+
+//#if os(Linux)
```

Comment:
@Lukasa I had it like this for testing but from what I read in libdispatch this won't be appropriate on Darwin: https://github.com/apple/swift-corelibs-libdispatch/blob/34f383d34450d47dd5bdfdf675fcdaa0d0ec8031/src/shims/time.h#L126-L159.

How should we proceed here? Should we (a) just do this Linux only for now and continue to just call `DispatchTime.now()` on Darwin, or (b) do something different on Darwin, e.g. `mach_absolute_time()`?